easybuild/easyconfigs/h/h-yade/h-yade-2017.01a-foss-2015b.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'CMakeMake'
+
+name = 'h-yade'
+version = '2017.01a'
+
+homepage = 'http://yade-dem.org/doc/index.html#'
+description = "Yade is an extensible open-source framework for discrete numerical models, focused on Discrete Element Method."
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+builddependencies = [('CMake', '3.1.3')]
+
+dependencies = [('LOKI', '0.1.7'),
+	        ('Qt', '4.8.5', '-GLib-2.44.0'),
+                ('libQGLViewer', '2.5.1'),
+                ('Boost', '1.58.0', '-Python-2.7.10'),
+                ('MPFR', '3.1.2', '-GMP-6.0.0a'),
+                ('Eigen', '3.2.1'),
+                ('GTS', '0.7.6'),
+                ('GL2PS', '1.3.8'),
+                ('PyGTS', '0.3.1', '-Python-2.7.10'),
+                ('PyQt', '4.11.4', '-Python-2.7.10'),
+		('IPython', '3.2.3', '-Python-2.7.10'),
+		('Python-Xlib', '0.15rc1', '-Python-2.7.10'),
+		('matplotlib', '1.5.1', '-Python-2.7.10'),
+		('PIL', '1.1.7', '-Python-2.7.10'),
+ 		('minieigen', '0.5.3', '-Python-2.7.10') ]
+
+
+sources = ['yade-%(version)s.tar.gz']
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+
+separate_build_dir = True
+
+preconfigopts = 'export EBINSTALLPREFIX=%(installdir)s && '
+preconfigopts += 'export CC=gcc && export CXX=g++ && '
+preconfigopts += 'cp /apps/leone/UES/RH6.7_PE15.12/HILTI/EBSandBox/brefcom-mm.hh %(builddir)s && '
+
+configopts = '-DINSTALL_PREFIX=$EBINSTALLPREFIX '
+configopts += '-DLOKI_INCLUDE_DIR=$EBROOTLOKI/include/ '
+configopts += '-DLOKI_LIBRARY=$EBROOTLOKI/lib/ '
+configopts += '-DENABLE_VTK=OFF '
+configopts += '-DENABLE_CGAL=OFF '
+configopts += '-DENABLE_LINSOLV=OFF '
+configopts += '-DENABLE_PFVFLOW=OFF '
+configopts += '-DBoost_NO_SYSTEM_PATHS=ON '
+configopts += '-DBoost_INCLUDE_DIR=$EBROOTBOOST/include '
+configopts += '-DBOOST_LIBRARYDIR=$EBROOTLOKI/lib '
+configopts += '-DBoost_NO_BOOST_CMAKE=ON '
+configopts += '-DGL2PS_LIBRARY=$EBROOTGL2PS/lib '
+configopts += '-DGL2PS_INCLUDE_DIR=$EBROOTGL2P/include '
+configopts += '-DEIGEN3_INCLUDE_DIR=$EBROOTEIGEN/include/ '
+configopts += '-DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python '
+configopts += '-DPYTHON_INCLUDE_DIR=$EBROOTPYTHON/include/python2.7 '
+configopts += '-DPYTHON_LIBRARY=$EBROOTPYTHON/lib '
+configopts += '-DQGLVIEWER_INCLUDE_DIR=$EBROOTLIBQGLVIEWER/include '
+configopts += '-DUSE_QT5=OFF '
+configopts += '-DCMAKE_CXX_FLAGS="-DYADE_CPM_FULL_MODEL_AVAILABLE" '
+
+modextravars = {'LD_PRELOAD':'/apps/leone/UES/RH6.7_PE15.12/HILTI/EBSandBox/easybuild/software/GL2PS/1.3.8-foss-2015b/lib/libgl2ps.so'}
```

Comment:
please try to comment out this line to see if this workaround is still needed in the upgraded OS.

If it does not work, be sure to update to the path to the latest installation (or we should find a way to expand the `$EBROOTGL2PS` variable here).